src/observer.ts

```diff
@@ -1,9 +1,10 @@
 import { forwardRef, memo } from "react"
 import { isUsingStaticRendering } from "./staticRendering"
-import { useObserver } from "./useObserver"
+import { ForceUpdateHook, useObserver } from "./useObserver"
 
 export interface IObserverOptions {
     readonly forwardRef?: boolean
+    readonly customForceUpdateHook?: ForceUpdateHook
```

Comment:
Let's not pollute the `observer` with this. It's a low-level API and it should be on `useObserver` only. You can make your own `observer` for those needs.

Reply:
The problem with that is basically that to make a custom observer would mean to copy and paste the whole thing anew :-/

Reply:
Yes, that's exactly that :) If you want to make crazy stuff you have to do something extra ;) Honestly, it's not such a big thing to copy.

Reply:
no, but then that'd mean that I'd need recopy and paste everytime observer changes and change one line :-/

Reply:
Sure, but on another hand, it's about the simplicity of the API. It's enough to allow this craziness in a low-level API, I don't like to spread more like that. Besides, I don't think that `observer` will be changing too much in the future, there is very little logic. The typings are the bulkiest part in here.

Reply:
removed from observer